server/mvcc/backend/tx_buffer.go

```diff
@@ -69,10 +71,19 @@ func (txw *txWriteBuffer) writeback(txr *txReadBuffer) {
 		rb.merge(wb)
 	}
 	txw.reset()
+	// mark read buffer modified
+	txr.modifiedVersion++
```

Comment:
Can we rename this `bufferRevision`? This seems confusing with modified revision we have for global index.

Reply:
> Can we rename this `bufferRevision`? This seems confusing with modified revision we have for global index.

in this case, shall we call it bufferVersion in case people getting confused with the data revision? or do you have any other suggestion?

Reply:
> Can we rename this `bufferRevision`? This seems confusing with modified revision we have for global index.

How about naming it sequence number?